test_tableslurp.sh

```diff
@@ -2,8 +2,30 @@
 
 # Script Name: test_tableslurp.sh
 
+# Purpose: This script is called by the Makefile which passes several arguments
+# and then runs tableslurp in the background, e.g., ./tableslurp& to download a
+# file from AWS S3 bucket, completing a backup process.
+
 # Author: Kim Lew
 
+# exit 1 - is a Catchall for general errors.
+usage() {
+	cat <<-EOF
+	USAGE: $(basename "$0") DIRECTORY AWS_ACC_KEY_ID AWS_SEC_ACC_KEY BUCKET
+	Downloads a file from the target directory, /tmp/tablesnap-test.
```

Comment:
Downloads a file from S3 to the test directory?

Reply:
Missed that. Thanks. Changed.